### ISSUE TITLE
searching on gene name  

### ISSUE THREAD

**Issue by ValWood** [geneontology/amigo#714]
Historically 
I used to search on gene name for pombe and I always used to find the entity.  These days, as more species are added I never can, because it's beneath the bottom of the list and there is no way to alter my search to improve the specificity:

<img width="1343" alt="Screenshot 2024-02-01 at 16 23 03" src="https://github.com/geneontology/amigo/assets/7359272/862af0a2-8291-427e-8da5-6e351339bc78">

Is there any way to  either
a) prioritise the  core model species
b) make the  results list longer and scrollable
c) ideally  always qualify by species, or you have no hope of finding what you need based on just the gene name and the uniprot ID

**Comment by kltm** [comment]
Yes, this is an increasing annoyance and is on our radar. While we try and get back to backend work, I can offer a couple of workarounds:

### Pre-pinned gene product search (RECOMMENDED)

If you're looking for gene products for your species, the "general" search on the landing page can get noisy. You can also go straight to a gp search for your organism at:

https://amigo.geneontology.org/amigo/search/bioentity?q=*:*&fq=taxon_subset_closure_label:%22Schizosaccharomyces%20pombe%22&sfq=document_category:%22bioentity%22

### Weird thing that works, but I'm not sure why (NOT RECOMMENDED)

On the landing page, you can type:

`pcm1 Pom` <- note the cap "P" in "Pom"

The search isn't supposed to be sensitive like this, especially not here, but is does seem to give the "correct" result for some reason.

**Comment by ValWood** [comment]
Thanks!
